pkg/operator/ceph/cluster/osd/spec.go

```diff
@@ -626,3 +627,24 @@ func (c *Cluster) getExpandPVCInitContainer(osdProps osdProperties, osdID string
 
 	return container
 }
+
+func (c *Cluster) getPauseContainerCommands(command []string, args []string) []string {
```

Comment:
```suggestion
func getPauseContainerCommands(command []string, args []string) []string {
```

Why is this a method?

Reply:
I planned to have this method because i do not wanted to clutter the read view of the main function.